frame/staking/src/benchmarking.rs

```diff
@@ -0,0 +1,399 @@
+		for i in 0 .. v {
+			invulnerables.push(account("invulnerable", i, SEED));
+		}
+	}: _(RawOrigin::Root, invulnerables)
+
+	force_unstake {
+		let u in ...;
+		let (stash, _) = create_stash_controller::<T>(u)?;
+	}: _(RawOrigin::Root, stash)
+
+	cancel_deferred_slash {
+		let s in 1 .. 1000;
+		let mut unapplied_slashes = Vec::new();
+		let era = EraIndex::one();
+		for _ in 0 .. 1000 {
+			unapplied_slashes.push(UnappliedSlash::<T::AccountId, BalanceOf<T>>::default());
+		}
+		UnappliedSlashes::<T>::insert(era, &unapplied_slashes);
+
+		let slash_indices: Vec<u32> = (0 .. s).collect();
+	}: _(RawOrigin::Root, era, slash_indices)
+
+	payout_validator {
+		let n in 1 .. MAX_NOMINATIONS as u32;
+		let validator = create_validator_with_nominators::<T>(n, MAX_NOMINATIONS as u32)?;
+		let current_era = CurrentEra::get().unwrap();
+	}: _(RawOrigin::Signed(validator), current_era)
+
+	payout_nominator {
+		let v in 0 .. MAX_NOMINATIONS as u32;
+		let (nominator, validators) = create_nominator_with_validators::<T>(v)?;
+		let current_era = CurrentEra::get().unwrap();
+		let find_nominator = validators.into_iter().map(|x| (x, 0)).collect();
+	}: _(RawOrigin::Signed(nominator), current_era, find_nominator)
+
+	rebond {
+		let l in 1 .. 1000;
+		let (_, controller) = create_stash_controller::<T>(u)?;
+		let mut staking_ledger = Ledger::<T>::get(controller.clone()).unwrap();
+		let unlock_chunk = UnlockChunk::<BalanceOf<T>> {
+			value: 1.into(),
+			era: EraIndex::zero(),
+		};
+		for _ in 0 .. l {
+			staking_ledger.unlocking.push(unlock_chunk.clone())
+		}
+		Ledger::<T>::insert(controller.clone(), staking_ledger);
+	}: _(RawOrigin::Signed(controller), (l + 100).into())
+
+	set_history_depth {
+		let e in 1 .. 100;
+		HistoryDepth::put(e);
+		CurrentEra::put(e);
+		for i in 0 .. e {
+			<ErasStakers<T>>::insert(i, T::AccountId::default(), Exposure::<T::AccountId, BalanceOf<T>>::default());
+			<ErasStakersClipped<T>>::insert(i, T::AccountId::default(), Exposure::<T::AccountId, BalanceOf<T>>::default());
+			<ErasValidatorPrefs<T>>::insert(i, T::AccountId::default(), ValidatorPrefs::default());
+			<ErasValidatorReward<T>>::insert(i, BalanceOf::<T>::one());
+			<ErasRewardPoints<T>>::insert(i, EraRewardPoints::<T::AccountId>::default());
+			<ErasTotalStake<T>>::insert(i, BalanceOf::<T>::one());
+			ErasStartSessionIndex::insert(i, i);
+		}
+	}: _(RawOrigin::Root, EraIndex::zero())
+
+	reap_stash {
+		let u in 1 .. 1000;
+		let (stash, controller) = create_stash_controller::<T>(u)?;
+		T::Currency::make_free_balance_be(&stash, 0.into());
+	}: _(RawOrigin::Signed(controller), stash)
+
+	new_era {
+		let v in 1 .. 10;
+		let n in 1 .. 100;
+		MinimumValidatorCount::put(0);
+		create_validators_with_nominators_for_era::<T>(v, n)?;
+		let session_index = SessionIndex::one();
+	}: { 
+		let maybe_validators = Staking::<T>::new_era(session_index).ok_or("`new_era` failed")?;
+		assert!(maybe_validators.len() == v as usize);
```

Comment:
```suggestion
		let validators = Staking::<T>::new_era(session_index).ok_or("`new_era` failed")?;
		assert!(validators.len() == v as usize);
```

(unwrapped option shall not be called _maybe_)